unlock-app/src/components/interface/checkout/main/Metadata.tsx

```diff
@@ -343,7 +343,7 @@ export function Metadata({ checkoutService, injectedProvider }: Props) {
           const address = await getAddressForName(item.recipient)
           return {
             ...item,
-            recipient: address,
+            recipient: (paywallConfig.recipient != undefined && paywallConfig.recipient != '') ? paywallConfig.recipient : address,
```

Comment:
For clarity please move that to a variable. Also, can you check if the `recipient` param in he paywall config can be added _inside_ the `lock` object, rather than just in the root?

Reply:
also check line 114 in this file :)

Reply:
> For clarity please move that to a variable. Also, can you check if the `recipient` param in he paywall config can be added _inside_ the `lock` object, rather than just in the root?

I tried to add it in the lock and the root and it doesn't work

Reply:
> also check line 114 in this file :)

yup, I check it before, but it only works for this: 

![image](https://github.com/unlock-protocol/unlock/assets/34518489/b2354cf0-ee30-4440-ae6c-e8f709cc6ff0)

Reply:
> For clarity please move that to a variable. Also, can you check if the `recipient` param in he paywall config can be added _inside_ the `lock` object, rather than just in the root?

I can Try like this... thinks it's better: 
![image](https://github.com/unlock-protocol/unlock/assets/34518489/d4bef7fb-85bc-437c-9e3c-e410baf1c710)

Reply:
> yup, I checked before, but it only works for this:

You should move what's on line 114 to the `Metadata` component...